### ISSUE TITLE
macos stat error

### ISSUE THREAD

**Issue by fawaf** [rfocosi/otp-cli#13]
running any command on macos will cause the following to print:
```
stat: illegal option -- -
usage: stat [-FLnq] [-f format | -l | -r | -s | -x] [-t timefmt] [file ...]
```

running anything that generates a token causes it to print twice

**Comment by rfocosi** [comment]
Hi! 
Which MacOs version are you using?

It prints an error and generate a token? Or just print the error?

**Comment by fawaf** [comment]
latest monterey, 12.3.1

it prints an error, but still generates a token

**Comment by rfocosi** [comment]
Fixed on 3.0.2 
If the issue persists, contact me again :)

Thank you for reporting!
